docs/api/covidcast-signals/hhs.md

```diff
@@ -0,0 +1,125 @@
+the sum of all adult and pediatric COVID-19 hospital admissions. This
+sum is used as the "ground truth" for hospitalizations by the (Reich
+lab COVID-19 forecast
+hub](https://github.com/reichlab/covid19-forecast-hub/blob/master/data-processed/README.md#hospitalizations].
+
+
+| Signal | Geography | Resolution | Description |
+| --- | --- |
+| `confirmed_admissions_1d` | state | 1 day | Sum of adult and pediatric confirmed COVID-19 hospital admissions occuring each day. |
+| `sum_confirmed_suspected_admissions_1d` | state | 1 day | Sum of adult and pediatric confirmed and suspected COVID-19 hospital admissions occuring each day. |
+| `confirmed_admissions_7d` | county, hrr, msa, state | 7 days | Sum of adult and pediatric confirmed COVID-19 hospital admissions occuring over a span of 7 days. |
+| `sum_confirmed_suspected_admissions_7d` | county, hrr, msa, state | 7 days | Sum of adult and pediatric confirmed and suspected COVID-19 hospital admissions occuring over a span of 7 days. |
+
+## Table of contents
+{: .no_toc .text-delta}
+
+1. TOC
+{:toc}
+
+## Estimation
+
+### Statewise, daily resolution
+
+Statewise daily resolution signals use the following four columns from
+the HHS state timeseries dataset:
+
+* `previous_day_admission_[adult|pediatric]_covid_[confirmed|suspected]`
+
+The `confirmed` signal is the sum of the two `confirmed` columns: 
+
+* adult
+* pediatric
+
+The `sum_confirmed_suspected` signal is the sum of all four columns:
+
+* adult confirmed
+* adult suspected
+* pediatric confirmed
+* pediatric suspected
+
+The source data specfies that admissions occured on the previous
+day. We automatically adjust the date of each result so that
+admissions are incident on that date.
+
+### Facilitywise, 7-day resolution
+
+Facilitywise 7-day resolution signals use the following four columns
+from the HHS facilitywise dataset:
+
+* `previous_day_admission_[adult|pediatric]_covid_[confirmed|suspected]_7_day_sum`
+
+The `confirmed` signal is the sum of the two `confirmed` columns: 
+
+* adult
+* pediatric
+
+The `sum_confirmed_suspected` signal is the sum of all four columns:
+
+* adult confirmed
+* adult suspected
+* pediatric confirmed
+* pediatric suspected
+
+The source data specfies that admissions for the previous day are
+summed over a seven-day period starting with the collection date. We
+automatically adjust the date for each result so that admissions are
+incident to the seven day period starting with that date.
+
+The source data uses an unusual week definition (originally
+Friday-Thursday; Thursday-Wednesday with our adjustment). Because this
+definition does not match that used by epiweeks, this dataset uses
+`time_type=day`, however data will only ever be present for dates that
+are the start of a seven-day collection period (ie Thursdays).
+
+Facilitywise data is aggregated to other geographies using the FIPS
+code of the facility, if available, and the ZIP5 code of the facility
+otherwise.
```

Comment:
Sentence restructure: "Facility-wise data is aggregated to other geographies using the facility's FIPS code, if available, and the ZIP5 code of the facility otherwise." 

Or keep Facilitywise if it's a title.

Reply:
```suggestion
Facility-wise data is aggregated to other geographies using the facility's FIPS
code, if available, and the ZIP5 code of the facility otherwise.
```